src/workloads/docs/CrudActorFSM.yml

```diff
@@ -0,0 +1,157 @@
+      # end all the actors will end their phases. If an actor is in a sleepBefore or sleepAfter
+      # delay, it will exit the phase. If the actor is executing a state's operations, it will
+      # complete those operations, and then exit the phase.
+      Operations:
+      - OperationName: updateOne
+        OperationCommand:
+          Filter: &filter {phoneID: {^ActorId: {}}}
+          Update:
+            $set: {currentPhoneState: "PhoneIsOn"}
+
+      # Transitions specifies next states, delays, and operations. The On state can transition to
+      # each of the three other states
+      Transitions:
+      - To: "Off"  # Transition to this state
+
+        # Transitions are probabilistically chosen based on weight. A transition with weight 2 is
+        # twice as likely to be picked as a transition with weight 1. Weights must be greater than
+        # 0.
+        Weight: 1
+
+        # Transitions also support the phase keywords of sleepBefore, sleepAfter, and repeat.  For
+        # sleepBefore, the system is considered to remain in it's current state. If there is a
+        # repeating action, it will continue repeating until the transition fires.  For repeat, the
+        # operation is repeated immediately.  sleepAfter will not be supported for transitions with
+        # no operations.
+
+        # ^TimeSpec is a proposed generator to randomly generate acceptable TimeSpecs with units. It
+        # accepts any valid int or double generator or constant for value.
+        SleepBefore: {^TimeSpec: {value: {^RandomDouble: {min: 0, max: 15}}, unit: "seconds"}}
+
+    - Name: "Off"  # The device is Off
+      Operations:
+      - OperationName: updateOne
+        OperationCommand:
+          Filter: *filter
+          Update:
+            $set: {currentPhoneState: "PhoneIsOff"}
+      Transitions:  # Can only go to On from Off
+      - To: "On"
+        Weight: 1
+        SleepBefore: {^TimeSpec: {value: {^RandomDouble: {min: 0, max: 15}}, unit: "seconds"}}
+
+    - Name: Sleep
+      Operations:
+      - OperationName: updateOne
+        OperationCommand:
+          Filter: *filter
+          Update:
+            $set: {currentPhoneState: "PhoneIsSleeping"}
+      Transitions:  # Can only go to On or Off from sleep
+      - To: "On"
+        Weight: 1
+        SleepBefore: {^TimeSpec: {value: {^RandomDouble: {min: 0, max: 15}}, unit: "seconds"}}
+      - To: "Off"
+        Weight: 1
+        SleepBefore: {^TimeSpec: {value: {^RandomDouble: {min: 0, max: 15}}, unit: "seconds"}}
+
+    - Name: Error
+      Operations:
+      - OperationName: updateOne
+        OperationCommand:
+          Filter: *filter
+          Update:
+            $set: {currentPhoneState: "PhoneIsInError"}
+      Transitions:  # Turn off on error
+      - To: "Off"
+        Weight: 1
+        SleepBefore: {^TimeSpec: {value: {^RandomDouble: {min: 0, max: 15}}, unit: "seconds"}}
```

Comment:
Not gonna block merge or anything, but fwiw most of the quotes in this example aren't necessary and (imho) are a bit distracting. We don't (yet) lint quotes, but that would be the first rule I'd add 😸

Reply:
FWIW, yamllint now passes on this file. It really wants On and Off quoted, otherwise it complains about truthy values should be true or false.